transports/webrtc/src/tokio/connection.rs

```diff
@@ -158,7 +181,7 @@ impl StreamMuxer for Connection {
             Some(detached) => {
                 log::trace!("Incoming substream {}", detached.stream_identifier());
 
-                let (substream, drop_listener) = Substream::new(detached);
+                let (substream, drop_listener) = Substream::new(detached, self.bandwidth.clone());
```

Comment:
Interesting, I thought you wanted to count the bytes on the UDP socket level? Measuring them here will not include the DTLS framing AFAIK.

Reply:
I did, but we don't have access to UDP socket here.

Reply:
Oh wait, we can pass it when creating connection. I will change the code.

Reply:
Nah, doing it involves passing udp_mux to connections, which we're trying to avoid altogether with udp_mux_handle. Probably not worth it.